openfoam-interface/libs/sourceFiles/LESModel.C

```diff
@@ -0,0 +1,207 @@
+/*---------------------------------------------------------------------------*\
```

Comment:
All OpenFoam files shipped in openfoam-interface/libs/sourceFiles have some necessary differences respect to the original ones (to avoid the use of input files).